locales/en.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
It looks like those paths are relative to 3akai-ux though, and this source doesn't end up in there.